examples/basic-china/tidb-cluster.yaml

```diff
@@ -0,0 +1,34 @@
+# IT IS NOT SUITABLE FOR PRODUCTION USE.
```

Comment:
Is it better to replace the `china` with something like `ali-registry`? It's strange to use `china` in the repo.

Reply:
how about `cn`? 

images we used in this example are located in China, I would suggest using a region name here.

Reply:
`ali-registry` is too specific

Reply:
ok